torch/_subclasses/functional_tensor.py

```diff
@@ -274,6 +277,10 @@ def cuda(self, device=None, *args, **kwargs):
     int = _conversion_method_template(dtype=torch.int32)
     long = _conversion_method_template(dtype=torch.int64)
 
+    @property
+    def layout(self):
+        return self.elem.layout
```

Comment:
It's weird you need to do this, I would have expected elem.layout passed to make subclass to work

Reply:
I am a bit surprised too (and it may be an issue in the future, but the PyTorch -> C++ interface seems to sometimes go directly to TensorImpl for this). And, not to hide behind others, @pearu suggested this fix ;-)